### ISSUE TITLE
tmp files not being cleared

### ISSUE THREAD

**Issue by csantero** [ef4/ember-auto-import#128]
I've noticed an issue in my projects using ember-auto-import: after killing `ember serve` with ctrl-c, there will be folders like `ember_auto_import_webpack-output_dir-Ar0BBq71.tmp` remaining behind under `tmp`. I've ensured that I started with a clean `tmp` to begin with.

**Comment by jrjohnson** [comment]
Related: I tried out `EMBER_CLI_BROCCOLI_2=true EMBER_CLI_SYSTEM_TEMP=true ember serve` today and I just wanted to report a few things that were placed in tmp instead of going to system-temp.

```
$ ls tmp
ember_auto_import_webpack-output_dir-M2kf7TSU.tmp
ember_auto_import_webpack-staging_dir-vOAPIcTQ.tmp
```

**Comment by rwjblue** [comment]
I _think_ that might be a different issue since the original poster is experiencing leaking temp dirs, whereas (AFAICT) the issue you are seeing @jrjohnson is that you expect `tmp` to be completely empty _during_ a build when using Broccoli@2. 

TLDR; I think that a refactor away from quick-temp (and to system temp) will fix the issue @jrjohnson is seeing in broccoli@2 mode, but the OP’s issue seems different (leaking tmp dirs after a completed build).

**Comment by josemarluedke** [comment]
Ember CLI v3.5 is now released and it contains Broccoli@2 with System temp enabled by default. I believe ember-auto-import should be revisited to move away from the `tmp` folder as well.
